File: Unidad 3/JAVA/P16_U3.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.p16_u3;

import java.util.Scanner;

/**
 *
 * @author LaboratorioLTI2
 */
public class P16_U3 {

    public static void main(String[] args) {
        Scanner sc = new Scanner (System.in);
        
        int c=1, hrs, pago, sueldo;
         
        while (c<=2){
            System.out.println("¿Cuantas horas trabajaste?");
            hrs = sc.nextInt();
            System.out.println("Ingresa el pago");
            pago = sc.nextInt();
            
            sueldo = pago * hrs;
            System.out.println("Su sueldo es " + sueldo);
            c++;
        }
        
    }
}

File: Unidad 3/C#/P16_U3.cs
using System;

namespace MyApp
{
    internal class Program
    {
        static void Main(string[] args)
        {
            int c = 1, hrs, pago, sueldo;

            while (c <= 20)
            {
                Console.WriteLine("¿Cuantas horas trabajaste?");
                hrs = Convert.ToInt32(Console.ReadLine());
                Console.WriteLine("Ingresa el pago");
                pago = Convert.ToInt32(Console.ReadLine());

                sueldo = pago * hrs;
                Console.WriteLine("Su sueldo es " + sueldo);
                c++;
            }
        }
    }
}